### DISCUSSION TITLE
Unlearn or hide Spells.

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#2707] [General]
As title says, the ability to hide or unlearn spells that are no longer useful to you. It's very straightforward on why this is nice for quality of life, as for coding it I have no idea. An additional step beyond the user unlearning a spell would be if mutation could take away spells.

**Comment by chaosvolt** [comment]
For the latter, you can do a lil bit of this depending on what the source of that spell is. If you rely on spells granted by mutations, then anything that removes that mutation will unlearn the spell, so mutations canceling each other can be used to disable access to certain spells.

A caveat there is that this works fine if the mutation is part of a mundane mutation category, and the only way to get that mutation category is via regular mutagen-type items. If it relies on a spell that grants that SPECIFIC mutation via the `MUTATE_TRAIT` flag however, there's no way to rig it so the mutations can supercede each other or otherwise ensure you can't reverse those steps and waffle between two mutually-exclusive spells.
